CODEOWNERS

```diff
@@ -0,0 +1,9 @@
+# This file provides an overview of code owners in this repository.
+
+# Each line is a file pattern followed by one or more owners.
+# The last matching pattern has the most precedence.
+# For more details, read the following article on GitHub: https://help.github.com/articles/about-codeowners/.
+
+# The default owners are automatically added as reviewers when you open a pull request unless different owners are specified in the file.
+
+* @fmvilas @derberg @github-actions[bot]
```

Comment:
```suggestion
* @fmvilas @derberg @jonaslagoni @github-actions[bot]
```

Reply:
but then we are not aligned with `spec` repo 😉

Reply:
so maybe let us get #130 and #128 done first and then we can add more folks, huh?

Reply:
Yeah, I'm happy to wait but I'd not worry about aligning with spec. JSON Schemas are not the spec but a partial representation of it. If we align here we'd have to align on parsers too, which I don't think makes sense. What do you think?

Reply:
so I know that we treat JSON Schema as a tool, but I was thinking about the alignment because it almost in 90% reflects the spec, not like in case of parser, and parser anyway depends on JSON Schema again in 90% for the validation.

if `spec` owners are not only gatekeepers for JSON Schema, in theory, it may happen that some validation rule can be changed here but not in spec. You know what I mean?

🤔

Reply:
> if spec owners are not only gatekeepers for JSON Schema, in theory, it may happen that some validation rule can be changed here but not in spec. You know what I mean?

Yeah, I get it, I just don't think it's very different than a parser implementing a custom validation that is not defined in the spec (like in https://github.com/asyncapi/spec/issues/650). Anyway, I'm not going to block this PR just because of that.

I'm approving and we can consider adding other people later 👍